### ISSUE TITLE
What are the minimum GPU specifications required to run the model?

### ISSUE THREAD

**Issue by hmnc0415** [KAIST-VICLab/cloud_nerf#23]
I want to run this model on my local machine. Could you provide me with the minimum GPU specifications needed? My local machine has a 2080 Ti

**Comment by KAIST-VICLab** [comment]
You can adjust the number of training rays in each iteration to fit your computational resource. I suggest also increasing the number of training iterations and decreasing the learning rate by whatever scale factor you decrease batch size.
